src/h2o_discovery/__init__.py

```diff
@@ -40,7 +41,9 @@ def discover(
         config_path: The path to the H2O CLI configuration file.
 
     Raises:
-        LookupError: If the URI cannot be determined.
+        exceptions.DiscoveryLookupError: If the URI cannot be determined.
+        exceptions.H2OCloudEnvironmentError: If there seems to be a problem with the
```

Comment:
nit: isn't this supposed to be `error.DiscoveryLookupError`? 🤔

Reply:
🤔

`error.DiscoveryLookupError` is one line above. It's being raised when we cannot determine the address of the server.

`exceptions.H2OCloudEnvironmentError` is raised when we have an address but received unexpected answer.